README.md

```diff
@@ -2,7 +2,7 @@
 
 ## How to create a new stack 
 
-1. in `stacks`, duplicate the entry you want to duplicate, then make modifications like docker image etc. 
+1. Goto `stacks` folder, duplicate the entry you want to create template, then make modifications like docker image etc. 
```

Comment:
the wording is not clearer to me. Feel free to rework these instructions so they match your process.